src/artm/messages.proto

```diff
@@ -387,6 +389,21 @@ message TopicKernelScore {
   optional StringArray topic_name = 10;
 }
 
+// Represents a configuration of a topic part in nwt score
+message TopicPartInNwtScoreConfig {
+  optional float eps = 1 [default = 1e-37];
+  optional string class_id = 2 [default = "@default_class"];
+  repeated string topic_name = 3;
+  repeated string topic_name_to_score = 4;
```

Comment:
Almost... I think ``topic_mass(i)`` should contain ``topic_part(i)`` instead of ``topic_part(i) / denominator``.